staff/francisco-fernandez/app-template/about.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About</title>
+</head>
+<body>
+        <h1>About Us</h1>
+        <p>We are a new young company with a lot of illusion and no idea of programming</p>
+        <footer>September 2018</footer>
```

Comment:
Inside ```<footer>```, try including a ```<time>```tag in order to administer better your data for future uses.